xdrip-get-entries.sh

```diff
@@ -546,6 +546,34 @@ function check_sensor_change()
     log "exiting"
     exit
   fi
+
+  curl --compressed -m 30 -H "API-SECRET: ${API_SECRET}" "${NIGHTSCOUT_HOST}/api/v1/treatments.json?find\[created_at\]\[\$gte\]=$(date -d "15 minutes ago" --iso-8601=seconds $UTC)&find\[eventType\]\[\$regex\]=Sensor.Stop" 2>/dev/null | grep "Sensor Stop"
```

Comment:
this seems like a reasonable temporary solution, and i like it, but.... its probably not what you intended for the long term because it means the typical 2-hour post-dated starts/stops aren't going to clear the cal cache...

otherwise lgtm.